### ISSUE TITLE
Documenting a component that is exported from another interface

### ISSUE THREAD

**Issue by tusharchutanigt** [styleguidist/react-styleguidist#1526]
We are making use of Material UI to develop our internal library but there are a few components that we want to use as is. So I am just doing the following in CircularIndicator.tsx
```export { CircularProgress as CircularIndicator } from '@material-ui/core';```

In the Readme.md when I try to test document the element I don't see anything but I do get this warning.

> Warning: ui/app/gtComponents/reactComponents/LoadingIndicator/CircularIndicator.tsx matches a pattern defined in “components” or “sections” options in your style guide config but doesn’t export a component.
> 
> It usually happens when using third-party libraries, see possible solutions here:
> https://react-styleguidist.js.org/docs/thirdparties.html

Looking at the doc I tried an annotation of /** @component */  but that doesn't work either. I also tried to import the component in Readme. Am I missing something here? 

EDIT: I am working in ts

**Comment by alexcanessa** [comment]
@tusharchutanigt were you able to solve this?

**Comment by mustafaKamal-fe** [comment]
> @tusharchutanigt were you able to solve this?

I have the same issue and have just yarn removed styleguidist !!
